WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Posts/BlogDashboardCardFrameView.swift

```diff
@@ -228,30 +194,17 @@ class BlogDashboardCardFrameView: UIView {
         ellipsisButton.setImage(UIImage.gridicon(.ellipsis).imageWithTintColor(.listIcon), for: .normal)
     }
 
-    private func updateChevronImageState() {
-        chevronImageView.isHidden = onViewTap == nil && onHeaderTap == nil
-        assertOnTapRecognitionCorrectUsage()
-    }
-
     private func updateEllipsisButtonState() {
         ellipsisButton.isHidden = onEllipsisButtonTap == nil
         let headerPadding = ellipsisButton.isHidden ?
             Constants.headerPaddingWithEllipsisButtonHidden :
             Constants.headerPaddingWithEllipsisButtonShown
         headerStackView.layoutMargins = headerPadding
-        assertOnTapRecognitionCorrectUsage()
-    }
-
-    /// Only one of two types of action should be associated with the card.
-    /// Either ellipsis button tap, or view/header tap
-    private func assertOnTapRecognitionCorrectUsage() {
-        let bothTypesUsed = (onViewTap != nil || onHeaderTap != nil) && onEllipsisButtonTap != nil
```

Comment:
These code removals from `BlogDashboardCardFrameView` are required for this change to work. Previously, setting both the `onHeaderTap` and `onEllipsisButtonTap` was unsupported. It is now.